### PR TITLE
tiny fix ... thanks jeremy

### PR DESCRIPTION
**Pull request by dmaclay**
not so tiny - but making it editable is now achieved by migrating it to the new form of forms for polls

### CODE REVIEW

**Comment by jerith** [comment]
:+1:

File: go/apps/multi_surveys/views.py [hunk 1]
```diff
@@ -161,14 +161,23 @@ def new_survey(request, conversation_key):
         'survey_form': survey_form,
     })
 
+def _clear_empties(cleaned_data):
+    """
+    FIXME:  this is a work around because for some reason Django is seeing
+            the new (empty) forms in the formsets as stuff that is to be
+            stored when it really should be discarded.
+    """
+    return [cd for cd in cleaned_data if cd.get('copy')]
```
Comment (hodgestar) [inline go/apps/multi_surveys/views.py]: If this is copied across from surveys, can we just import it?

Comment (dmaclay) [inline go/apps/multi_surveys/views.py]: done

File: go/apps/multi_surveys/views.py [hunk 2]
```diff
@@ -161,14 +162,15 @@ def new_survey(request, conversation_key):
...
 
-
 @login_required
 def survey(request, conversation_key, poll_name):
     conversation = conversation_or_404(request.user_api, conversation_key)
     poll_id = generate_poll_id(conversation, poll_name)
     pm, config = get_poll_config(poll_id)
-    if request.method == 'POST':
+    questions_data = config.get('questions', [])
+    completed_response_data = config.get('survey_completed_responses', [])
```
Comment (hodgestar) [inline go/apps/multi_surveys/views.py]: Maybe these could be moved down into the `elif` branch they're used in? Not too serious -- just took me a little while to find them.

Comment (dmaclay) [inline go/apps/multi_surveys/views.py]: done

**Comment by hodgestar** [comment]
Code looks good. Are there tests we can write to show the issue? Do the current tests pass?

**Pull request merged**